keps/sig-windows/2578-windows-conformance/README.md

```diff
@@ -0,0 +1,431 @@
+Kubernetes ecosystem.
+
+How will security be reviewed, and by whom?
+
+How will UX be reviewed, and by whom?
+
+Consider including folks who also work outside the SIG or subproject.
+-->
+
+## Design Details
+
+### Specification of Windows Operational Readiness (Kubernetes 1.23)
+
+- In order to qualify as a Windows test, the pods that demonstrate functionality for a given attribute of a cluster must be running on a `pause` image. 
+
+Additionally, this image must be scheduled to an operating system that meets the following requirements:
+
+- Windows Build Versions:
+  - Windows Server 2019 (LTSC)
+  - Windows Server 2022 (LTSC)
+
+- Installed KBs:
+  <!-- - Needs further discussion/investigation -->
+
+- Windows Hosts and containers must be running a patch level that is no older than 3 months
+
+- All tests in the Core Windows Operational Readiness suite depend on `pause`, and no other images.
+- All tests should use the `Pod.OS` specification field as the *only* required scheduling directive for pods, once this field is made GA.  Up until this time, the way tests schedule Windows pods can use RuntimeClasses, Labels, or taints in any way which "works" to implement the other requirements.
+
+#### Basic Networking
+
+- Ability to mount host volumes
+- Ability to access Linux container IPs *by service IP* from Windows containers
+- Ability to access Windows container IPs *by service IP* from Linux containers
+- Ability to access Linux container IPs *by NodePort IP* from Windows containers
+- Ability to access Windows container IPs *by NodePort IP* from Linux containers
+- Ability to access Linux container IPs *by pod IP* from Windows containers
+- Ability to access Windows container IPs *by pod IP* from Linux containers
+- Ability to mount a hostPath storage volume from a Kubelet into a pod and write to it. 
+- Ability to mount a hostPath storage volume from a Kubelet into a pod and read from it. 
+- Ability to schedule multiple containers, with distinct IP addresses, on the same node.
+- Ability to reboot all of the Windows nodes in a cluster while running Windows pods, in such a way that recovery of these deleted pods occurs in the scope of a few minutes.
+- Ability to delete and recreate services in such a way that load balancing rules for pods are recovered by whatever chosen service proxy is being utilized.
+- Ability to delete and recreate pods for StatefulSets which preserve their ability to serve as routed endpoints for services, while also having unchanging IP addresses.
+- Ability to access internal pods by internal Kubernetes DNS service endpoints.
+- Ability to access external services by Kubernetes DNS services endpoints (for non-airgapped clusters).
+- Ability to access internal pods through custom DNS records, injected by the Kubernetes pod Specification.
+- Ability to route services from pods from the EndpointSlice API.
+- Ability to run IPv4/IPv6 dual-stack networking (on supported OS Versions)
```

Comment:
dual-stack is not support in 2019 and there are no plans to backport it as of now.

Reply:
fixed